test/test_integration.py

```diff
@@ -13,55 +13,6 @@
 from opensquirrel.ir import Measure
 
 
-def test_simple_circuit() -> None:
-    qc = Circuit.from_string(
-        """
-        version 3.0
-
-        qubit[2] q
-        bit[2] b
-
-        H q[0]
-        CNOT q[0], q[1]
-
-        b = measure q
-        """
-    )
-    qc.replace(
-        CNOT,
-        lambda control, target: [
-            H(target),
-            CZ(control, target),
-            H(target),
-        ],
-    )
-
-    # Merge single-qubit gates and decompose with McKay decomposition.
-    qc.merge_single_qubit_gates()
-    qc.decompose(decomposer=McKayDecomposer())
-    assert (
-        str(qc)
-        == """version 3.0
-
-qubit[2] q
-bit[2] b
-
-Rz(1.5707963) q[0]
-X90 q[0]
-Rz(1.5707963) q[0]
-Rz(1.5707963) q[1]
-X90 q[1]
-Rz(1.5707963) q[1]
-CZ q[0], q[1]
-b[0] = measure q[0]
-Rz(1.5707963) q[1]
-X90 q[1]
-Rz(1.5707963) q[1]
-b[1] = measure q[1]
-"""
-    )
-
-
 def test_Spin2_backend() -> None:
     qc = Circuit.from_string(
```

Comment:
For the integration they used `Circuit.from_string`, so that's why it is also done here. But now they actually use both, so we should update our test to also use the circuit builder and see if the same result is produced. Should we add that to this ticket, or create another ticket for it?

Reply:
Another ticket I would say. Maybe there is an ongoing one that does exactly that.